### PR TITLE
osjs-monster-adapter

### PR DESCRIPTION
**Pull request by miladhashemi**
Add adapter github address in README.md

We have developed osjs-monster-adapter package for OS.js. 
https://www.npmjs.com/package/@burna/osjs-monster-adapter
https://github.com/BurnaSmartLab/osjs-monster-adapter

We want our package included to OS.js as an adapter such as webdav or FTP.

### CODE REVIEW

File: src/v3/resource/contrib/README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ If you want to submit your contribution, contact me by email, join the [Gitter](
 ## Filesystem
 
 * https://github.com/SpaceboyRoss01/osjs-devices-adapter - Adds all hardware devices to a root mountpoint
+* https://github.com/BurnaSmartLab/osjs-monster-adapter - An Adapter for os.js to work with swift saio object storage.
```
Comment (andersevenrud) [inline src/v3/resource/contrib/README.md]: Reword this to something like: `Swift saio object storage VFS adapter`

**Review by andersevenrud — CHANGES_REQUESTED** [review]
Also, update the commit message to reflect what was changed, not where.

**Review by miladhashemi — COMMENTED** [review]
Description changed to "Swift saio object storage VFS adapter"

**Review by miladhashemi — COMMENTED** [review]
Description updated.

**Comment by andersevenrud** [comment]
I'll just fix the commit message myself. I have to squash the commits anyways.

**Review by andersevenrud — APPROVED** [review]

**Comment by andersevenrud** [comment]
Thanks!

For reference, this is how I described the commit: https://github.com/os-js/manual.os-js.org/commit/40c3c4a2089b762c5d258c3fa4bcb0ee968569c8

**Comment by miladhashemi** [comment]
> Thanks!
> 
> For reference, this is how I described the commit: [40c3c4a](https://github.com/os-js/manual.os-js.org/commit/40c3c4a2089b762c5d258c3fa4bcb0ee968569c8)

Your welcome

Yes It's really better commit message! Certainly I will follow your commit message for my later commits.
Thanks!
